schemas/com.snowplowanalytics.monitor/application_status/jsonschema/1-0-0

```diff
@@ -0,0 +1,133 @@
+    "periodEnd": {
+      "description": "The end of the period to which this status applies",
+      "type": ["string", "null"],
+      "format": "date-time"
+    },
+    "inAppStats": {
+      "description": "Application execution and progress statistics",
+      "type": ["array", "null"],
+      "items": {
+        "description": "Each item contains stats for a specific tracker version",
+        "type": "object",
+        "properties": {
+          "vTracker": {
+            "description": "The tracker version",
+            "type": "string"
+          },
+          "minCollectorTstamp": {
+            "description": "The minimum collector timestamp processed in the period to which this status applies",
+            "type": ["string", "null"],
+            "format": "date-time"
+          },
+          "maxCollectorTstamp": {
+            "description": "The maximum collector timestamp processed in the period to which this status applies",
+            "type": ["string", "null"],
+            "format": "date-time"
+          },
+          "minLatency": {
+            "description": "The minimum data latency in the period to which this status applies, calculated as difference between collector_tstamp and now(); in milliseconds",
+            "type": ["integer", "null"]
+          },
+          "avgLatency": {
+            "description": "The average data latency in the period to which this status applies, calculated as difference between collector_tstamp and now(); in milliseconds",
+            "type": ["integer", "null"]
+          },
+          "maxLatency": {
+            "description": "The maximum data latency in the period to which this status applies, calculated as difference between collector_tstamp and now(); in milliseconds",
+            "type": ["integer", "null"]
+          },
+          "numEventsTotal":{
+            "description": "The total number of events processed in the period to which this status applies",
+            "type": ["integer", "null"]
+          },
+          "numEventsSucceeded":{
+            "description": "The number of successfully processed events in the period to which this status applies",
+            "type": ["integer", "null"]
+          },
+          "numEventsFailed":{
+            "description": "The number of events that failed to be processed in the period to which this status applies",
+            "type": ["integer", "null"]
+          }
+        },
+        "required": ["vTracker"],
+        "additionalProperties": false
+      }
+    },
+    "numNodes": {
+      "description": "The number of nodes (instances, containers, pods, etc) on which the application is running in the period to which this status applies",
+      "type": ["integer", "null"]
+    },
+    "nodeType": {
+      "description": "The type of nodes being used, e.g. EC2, Dataflow, Pod, etc",
+      "type": ["string", "null"]
+    },
+    "nodeSpecs": {
+      "description": "Detailed specifications for the nodes being used",
```

Comment:
@jbeemster - the way @dilyand has designed this schema allows for different node types and specification, whereas when we were brainstorming we assumed they would all be of the same type.

Is the approach @dilyand has taken safer? Or can we assume that the node types within an application will always be the same. (In which case this is over-engineering?)